pkg/kritis/review/validating_transport.go

```diff
@@ -101,51 +103,74 @@ func (avt *AttestorValidatingTransport) validatePublicKeyId(pubKey v1beta1.Publi
 	return nil
 }
 
+// NOTE: I am considering changing this function signature to:
+// func (avt *AttestorValidatingTransport) CheckValidatedAttestation(image string) error
+// The function would propagate the error (or success) of the
+// verifier.VerifyAttestation call. It currently returns ValdiatedAttestations,
+// which contain unused and not particularly meaningful information.
+// This is not a trivial change, and would be done in a follow-up PR.
 func (avt *AttestorValidatingTransport) GetValidatedAttestations(image string) ([]attestation.ValidatedAttestation, error) {
-	keys := map[string]v1beta1.PublicKey{}
+	publicKeys := []cryptolib.PublicKey{}
 	numKeys := len(avt.Attestor.Spec.PublicKeys)
-	for i, pubKey := range avt.Attestor.Spec.PublicKeys {
-		if err := avt.validatePublicKey(pubKey); err != nil {
+	validatedAtts := []attestation.ValidatedAttestation{}
+
+	for i, attestorKey := range avt.Attestor.Spec.PublicKeys {
```

Comment:
I suggest factoring this into a helper function "parsePublicKeys" or something such.

Reply:
Done.